src/config.py

```
@@ -100,7 +100,7 @@
 
 # Headers for requests https://www.whatismybrowser.com/guides/the-latest-user-agent/chrome
 headers = {
-    "User-Agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/142.0.0.0 Safari/537.36",  # noqa: E501
+    "User-Agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/144.0.0.0 Safari/537.36",  # noqa: E501
```

Ruff (0.14.11)

Unused `noqa` directive (non-enabled: `E501`)

Remove unused `noqa` directive

(RUF100)